doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-con.adoc

```diff
@@ -0,0 +1,7 @@
+[id='business-central-security-management-con_{context}']
+
+= Security management
+
+Security management is the process of managing users, groups, and permissions. You can control the access to {CENTRAL} resources and features from the {CENTRAL} Security management page.
+
+{CENTRAL} defines three types of entities for security management: users, groups, and roles. You can assign permissions to both roles and groups. Users inherit permissions from the groups and roles that the user is a member of.
```

Comment:
...roles that they are a member of...
or
A user inherits permissions from the groups and roles that the user is a member of.

Reply:
Done